### PR TITLE
Fix RuntimeError: Cannot join current thread

### PR DESCRIPTION
**Pull request by DarkCeptor44**
Fix for #61.

### CODE REVIEW

**Comment by abrahammartin** [comment]
this fix works for me as well.

File: LGTV/remote.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def __execute(self):
             method = list(command.keys())[0]
             args = command[method]
             self.__class__.__dict__[method](self, **args)
-        self.__waitClose()
```
[the remark this answers is not in the document]
Comment (DarkCeptor44) [reply LGTV/remote.py:115]: I'm also not sure of why it's happening but letting that method run causes the error eventually, by simply commenting it out there's enough time for the response to come back and not cause a `socket timeout` error.

**Comment by Gustry** [comment]
This fix is indeed working for me as well.

**Comment by RiaanLab** [comment]
Yip, fix worked as promised. Thanks.

**Comment by arkonbob** [comment]
Works for me too - not sure if these spammy comments are the way to go or just liking @DarkCeptor44 's original "Fix" comment, so went with both ;-)

**Pull request merged**